### PR TITLE
domain ids

### PR DESCRIPTION
**Pull request by lucyjemutai**
This pr fixes the display of the concept domain

### CODE REVIEW

File: pages/api/domains/domains.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     },
     {
         "id": "procedures",
-        "name": "Procedures and Services",
+        "name": "Medical Procedures",
```
Comment (alaboso) [inline pages/api/domains/domains.json:33]: should it be `medical-procedures`?

File: pages/index.js [hunk 2]
```diff
@@ -42,19 +42,19 @@
             },
             { id: "investigations", name: "Investigations (Lab and Imaging)", icon: "microscope", "metadata": { number_of_subdomains: 23, number_of_concepts: 7231 } },
             {
-                id: "procedures-services",
+                id: "procedures",
```
Comment (alaboso) [inline pages/index.js:45]: should it be medical-procedures?

**Review by alaboso — CHANGES_REQUESTED** [review]

File: pages/api/domains/domains.json (hunk 1 above)
Comment (lucyjemutai) [reply pages/api/domains/domains.json:33]: yes

File: pages/index.js (hunk 2 above)
Comment (lucyjemutai) [reply pages/index.js:45]: yes

**Pull request merged**